core/scss/elements/typography/_typography.scss

```diff
@@ -87,6 +87,10 @@ dfn {
   @include slab;
 }
 
+.su-script {
+  @include script;
```

Comment:
I did a bit of checking and I cannot find anything that would say that `script` is a reserved language keyword. As this may be confused with scripting files perhaps we could come up with a slightly different name?

Reply:
Changed to `handwriting`.